mentors/clojure/j.json

```
@@ -1,4 +1,12 @@
 [
+    {
```

```suggestion
  {
```
`{` should be indented to the same level as `}`.